src/main/java/com/internetitem/logback/elasticsearch/writer/ElasticsearchWriter.java

```diff
@@ -86,15 +108,29 @@ public void sendData() throws IOException {
 			urlConnection.disconnect();
 		}
 
+		updateBuffersOnSuccess();
+	}
+
+	public void clear() {
+		updateBuffersOnSuccess();
```

Comment:
This'll discard the data in the send buffer and move on with the contents of the backupBuffer (possibly empty)